contracts/contracts/FillManager.sol

```diff
@@ -22,7 +22,7 @@ contract FillManager is Ownable, LpWhitelist {
     ///
     /// .. seealso:: :sol:func:`fillRequest`
     event RequestFilled(
-        uint256 indexed requestId,
+        bytes32 indexed requestId,
```

Comment:
I suggest to consider using an `uint256` representation for request IDs, i.e. `uint256 request_id = uint256(hash(...))`. No data would be lost and it would be easier to handle than a 32-byte sequence.

Reply:
I would rather do that on the consumer side of the event.